source/features/list-pr-for-branch.tsx

```diff
@@ -49,7 +48,8 @@ async function init(): Promise<void | false> {
 			data-hovercard-type="pull_request"
 			data-hovercard-url={prInfo.url + '/hovercard'}
 		>
-			<StateIcon className={stateColorMap[prInfo.state]}/> #{prInfo.number}
+			<StateIcon className={stateColorMap[prInfo.state]}/>
+			<span> #{prInfo.number}</span>
```

Comment:
This allows the icon right margin to correctly apply:

![image](https://user-images.githubusercontent.com/44045911/132730469-d1fcc08a-e649-46a3-8fde-b55451941491.png)